src/Common/Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>net452</TargetFramework>
+		<TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
```

Comment:
this needs to target net452 explicitly as other net452 projects can't reference net standard 2.0 libs directly.